platforms/documentation/docs/build.gradle.kts

```diff
@@ -1,98 +1,110 @@
     // The 'gradlebuild.generate-samples' plugin uses the 'org.gradle.samples' plugin from the old gradle/guides build, which pulls in slf4j-simple, which we don't want.
     // Because this is done directly by the plugin application logic, we can't use a ComponentMetadataRule to exclude it.
     // See: https://github.com/gradle/guides/blob/ba018cec535d90f75876bfcca29381d213a956cc/subprojects/gradle-guides-plugin/src/main/java/org/gradle/docs/samples/internal/SamplesDocumentationPlugin.java#L335
-    exclude([group: "org.slf4j", module: "slf4j-simple"])
+    exclude("org.slf4j", "slf4j-simple")
 }
 
 dependencies {
     // generate Javadoc for the full Gradle distribution
-    runtimeOnly project(":distributions-full")
+    runtimeOnly(project(":distributions-full"))
 
-    userGuideTask 'xalan:xalan:2.7.1'
-    userGuideTask 'xerces:xercesImpl:2.11.0'
-    userGuideTask 'net.sf.xslthl:xslthl:2.0.1'
+    userGuideTask("xalan:xalan:2.7.1")
+    userGuideTask("xerces:xercesImpl:2.11.0")
+    userGuideTask("net.sf.xslthl:xslthl:2.0.1")
 
-    userGuideStyleSheets 'net.sf.docbook:docbook-xsl:1.75.2:resources@zip'
+    userGuideStyleSheets("net.sf.docbook:docbook-xsl:1.75.2:resources@zip")
 
-    jquery "jquery:jquery.min:3.5.1@js"
+    jquery("jquery:jquery.min:3.5.1@js")
 
-    testImplementation project(":base-services")
-    testImplementation project(":core")
-    testImplementation libs.jsoup
-    testImplementation "org.gebish:geb-spock:2.2"
-    testImplementation 'org.seleniumhq.selenium:selenium-htmlunit-driver:2.42.2'
-    testImplementation libs.commonsHttpclient
-    testImplementation libs.httpmime
+    testImplementation(project(":base-services"))
+    testImplementation(project(":core"))
+    testImplementation(libs.jsoup)
+    testImplementation("org.gebish:geb-spock:2.2")
+    testImplementation("org.seleniumhq.selenium:selenium-htmlunit-driver:2.42.2")
+    testImplementation(libs.commonsHttpclient)
+    testImplementation(libs.httpmime)
 
-    docsTestImplementation platform(project(":distributions-dependencies"))
-    docsTestImplementation project(":internal-integ-testing")
-    docsTestImplementation project(":base-services")
-    docsTestImplementation project(":logging")
-    docsTestImplementation libs.junit5Vintage
-    docsTestImplementation libs.junit
+    docsTestImplementation(platform(project(":distributions-dependencies")))
+    docsTestImplementation(project(":internal-integ-testing"))
+    docsTestImplementation(project(":base-services"))
+    docsTestImplementation(project(":logging"))
+    docsTestImplementation(libs.junit5Vintage)
+    docsTestImplementation(libs.junit)
 
-    integTestDistributionRuntimeOnly project(":distributions-full")
+    integTestDistributionRuntimeOnly(project(":distributions-full"))
 }
 
 asciidoctorj {
-    version = '2.5.13'
-    modules.pdf.version '2.3.10'
+    setVersion("2.5.13")
+    modules {
+        pdf.use()
+        pdf.setVersion("2.3.10")
+    }
```

Comment:
❌  Why do we call `.use()` ? What it does is sets the version to some default, but then we immediately override it.
We should be fine with just setting the version directly

```
    setVersion("2.5.13")
    modules.pdf.setVersion("2.3.10")
```